lib/config-generator.js

```diff
@@ -185,6 +186,13 @@ class ConfigGenerator {
             });
         }
 
+        if (this.webpackConfig.useStylusLoader) {
+            rules.push({
+                test: /\.stylus/,
```

Comment:
Is the `.stylus` file extension the recommended one? I saw `.styl` used on multiple pages:

* [Stylus Github](https://github.com/stylus/stylus)
* [Stylus documentation](http://stylus-lang.com/docs/#basic-usage)
* [Stylus-loader Github](https://github.com/shama/stylus-loader).

Reply:
you're absolutely right, that is a typo, which i'll fix right away :)